src/EventStore.Core.Tests/options.cs

```diff
@@ -79,33 +82,40 @@ class FakeConfigurationRoot : IConfigurationRoot {
 			CollectionAssert.AreEquivalent(expected, actual);
 		}
 
+		private static IEnumerable<KeyValuePair<string, object>> SpecificOrDefaultValues(List<KeyValuePair<string, object>> specificValues) {
+			var defaultValues = ClusterVNodeOptions.DefaultValues;
+			if (specificValues is null || specificValues.IsEmpty()) return defaultValues;
+			var specifiedKeys = specificValues.Select(kvp => kvp.Key).ToList();
+			return defaultValues.Where(kvp => !specifiedKeys.Contains(kvp.Key)).Concat(specificValues);
+		}
+
 		public static IEnumerable<PrecedenceCase> PrecedenceCases() {
 			yield return new(
 				new Dictionary<string, object> {[nameof(ClusterVNodeOptions.Application.StatsPeriodSec)] = 1},
 				string.Empty,
 				new Dictionary<string, string>(),
-				Array.Empty<string>(),
+				new[] { "--insecure" },
```

Comment:
Is it still necessary to set the insecure flag in these tests after the most recent changes?

Reply:
Removed it.